includes/Story_Renderer.php

```diff
@@ -236,4 +234,28 @@ public function render() {
 		$markup = $this->replace_body_end_tag( $markup );
 		return $markup;
 	}
+
+	/**
+	 * Get story meta images.
+	 *
+	 * There is a fallback poster-portrait image added via a filter, in case there's no featured image.
+	 *
+	 * @param int|\WP_Post|null $post Post.
+	 * @return string[] Images.
+	 */
+	public function get_story_meta_images( $post = null ) {
```

Comment:
Can this be kept `protected`? `$post` can then be _required_.

Reply:
This one was a copy and paste from Media class. But I will make the changes.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/2623/commits/f09fe8405bc103857762050794b506a3657fead8

Reply:
Fixed.